src/main/java/com/arcbees/bourseje/client/admin/dashboard/AdminDashboardView.java

```diff
@@ -52,7 +52,7 @@
     @UiField
     Element currentState;
     @UiField
-    HTMLPanel candidates;
+    SpanElement candidates;
```

Comment:
If this is to contain other elements, i think it should be a div instead. Putting block elements inside inline elements is usually not a good idea.